src/v2i-hub/MessageLoggerPlugin/src/MessageLoggerPlugin.cpp

```diff
@@ -268,11 +268,19 @@ void MessageLoggerPlugin::HandleBasicSafetyMessage(BsmMessage &msg, routeable_me
 			if (bsm->partII[0].list.count >= BSMpartIIExtension__partII_Value_PR_SpecialVehicleExtensions ) {
 				try
 				{
+					#if SAEJ2735_SPEC >= 2024
+					if(bsm->partII[0].list.array[1]->partII_Value.choice.SpecialVehicleExtensions.doNotUse !=NULL){
+						cJSON_AddItemToObject(_BsmMessageContent, "trailerPivot", cJSON_CreateNumber(bsm->partII[0].list.array[1]->partII_Value.choice.SpecialVehicleExtensions.doNotUse->connection.pivotOffset));
+						cJSON_AddItemToObject(_BsmMessageContent, "trailreLength", cJSON_CreateNumber(bsm->partII[0].list.array[1]->partII_Value.choice.SpecialVehicleExtensions.doNotUse->units.list.array[0]->length));
+						cJSON_AddItemToObject(_BsmMessageContent, "trailerHeight", cJSON_CreateNumber(bsm->partII[0].list.array[1]->partII_Value.choice.SpecialVehicleExtensions.doNotUse->units.list.array[0]->height[0]));
```

Comment:
![image](https://github.com/user-attachments/assets/5c1b87e9-e20c-46cf-ad27-b391f2cce908)
![image](https://github.com/user-attachments/assets/f48f14ad-067a-4236-b048-365c49e1271f)
According to J2735 2024 carma specs, the vehicleHeight is not an array nor list. Do we need `height[0]` rather than `height`?

Reply:
Fixed